core/src/main/java/io/questdb/griffin/engine/orderby/RecordTreeChain.java

```diff
@@ -376,6 +386,9 @@ public void toTop() {
         }
 
         private void of(RecordCursor base) {
+            if (!isOpen) {
```

Comment:
nit: redundant if

Reply:
Initially I wanted to keep things consistent with what we have in cursor record factories, but I agree that it's redundant. Removed in 492dfb943175c6fc952f26abb83d06524d20c6b5